src/js/environment/environment.js

```diff
@@ -158,3 +165,10 @@ Object.defineProperties(Features, {
         enumerable: true
     }
 });
+
+Object.defineProperties(Hardware, {
```

Comment:
Please put `touch` in `Features`. I think we should also add `mouse`, so touch-only is `Features.touch && !Features.mouse`.

Reply:
Even if we do that, it doesn't map to the checks we use in most of our UI https://github.com/jwplayer/jwplayer/blob/ed44063687a787c7e6daf5b8fe19fd240909fb85/src/js/utils/ui.js#L6